components/Banner.tsx

```diff
@@ -1,3 +1,4 @@
+"use client";
```

Comment:
While you're not using any client side rendering functionality, please remove `"use client"`.

Reply:
use client it's necessary otherwise I have a framer motion error